src/domain/individualApplication/IndividualApplicationPage.tsx

```diff
@@ -105,19 +110,23 @@ const IndividualApplicationPage: React.SFC<IndividualApplicationPageProps> = ({
 
       {similarCustomersData && (
         <>
-          <Notification labelText={t('individualApplication.noCustomerProfileNotification.label')} type="warning">
-            {t('individualApplication.noCustomerProfileNotification.description')}
-          </Notification>
+          <div className={styles.fullWidth}>
+            <Notification labelText={t('individualApplication.noCustomerProfileNotification.label')} type="warning">
+              {t('individualApplication.noCustomerProfileNotification.description')}
+            </Notification>
+          </div>
           <Table
             className={styles.fullWidth}
             data={similarCustomersData}
+            loading={loadingCustomers}
             columns={columns}
             renderMainHeader={() => t('individualApplication.customersTable.mainHeader')}
             renderTableToolsTop={({ selectedRows }) => {
               const onLinkCustomer = selectedRows.length ? () => handleLinkCustomer(selectedRows[0].id) : undefined;
 
               return <CustomersTableTools {...customerTableTools} handleLinkCustomer={onLinkCustomer} />;
             }}
+            renderTableToolsBottom={() => <Pagination {...pagination} className={styles.fullWidth} />}
```

Comment:
Would it make sense to render pagination as `renderTableToolsBottom` in other places also?

Reply:
Makes sense.
done.